python/cuml/cuml_accel_tests/upstream/umap/run-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
+# This script runs the umap tests with the cuml.accel plugin.
+# Any arguments passed to this script are forwarded directly to pytest.
+#
+# Example usage:
+#   ./run-tests.sh                     # Run all tests
+#   ./run-tests.sh -v -k test_foo      # Run specific test with verbosity
+#   ./run-tests.sh -x --pdb            # Stop on first failure and debug
+
+set -eu
+
+UMAP_TAG="release-0.5.7"
```

Comment:
I wonder if there's a way to fetch this version, e.g. generating it from dependencies.yaml or fetching from pyproject.toml, so that we don't have to maintain it separately here.

Reply:
I thought about that, but decided to skip it for now. We currently pin to only a single umap version, and umap is released so infrequently. They also don't have the most consistent release tag naming scheme. I think if/when we decide to expand umap versions we can reconsider this.